src/include/optimizer/binding.h

```diff
@@ -10,7 +10,6 @@
 //
 //===----------------------------------------------------------------------===//
 
-
 #pragma once
```

Comment:
I didn't notice when the binder code checked in, but please add underscore for all the member variables for this class as well. This will make other people later work on the code base easy to understand your code. You can use Eclipse to easily refactor variable name.